src/api/admin_user/handlers/edit_admin_user_handler.rs

```diff
@@ -25,10 +25,15 @@ pub async fn edit_admin_user_handler(
         .admin_user_service
         .find_by_id(&state.db, admin_user_id)
         .await?;
+    let success_message = session
```

Comment:
Fetching and display logic will go into an `admin_user_table_handler.rs` which already has logic so ideally it should work without any changes